Tests/TuistKitTests/Services/TestServiceTests.swift

```diff
@@ -265,15 +265,17 @@ private extension TestService {
         configuration: String? = nil,
         path: AbsolutePath,
         deviceName: String? = nil,
-        osVersion: String? = nil
+        osVersion: String? = nil,
+        skipUiTests: Bool = false
```

Comment:
Nit: There is inconsistent usage of `skipUiTests` and `skipUITests` - can you just do a quick search and replace, so only `skipUITests` is used?

Reply:
Fixed 💪